packages/native-stack/src/views/NativeStackView.native.tsx

```diff
@@ -341,7 +344,6 @@ const SceneView = ({
       homeIndicatorHidden={autoHideHomeIndicator}
       hideKeyboardOnSwipe={keyboardHandlingEnabled}
       navigationBarColor={navigationBarColor}
-      // @ts-expect-error prop supported from react-native-screens 3.32.0 onwards
```

Comment:
@satya164 This one caused the error. Not the one I added

Reply:
@maksg this is because the `@ts-expect-error` affects all of the props after it and all other errors are suppressed. so there can be only one `@ts-expect-error` for the props.

Reply:
Ah, that explains it then. Thanks!